recipes/alm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ALM" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "a3ee1b25a897ec8bd576dd00c948657c4faefbd42f2616bf5ad2ce79a3d8cda4" %}
```

Comment:
Move the sha256 to L11 for consistency with the new example recipe.

Reply:
Done